### DISCUSSION TITLE
What does `self.graph` do here?

### DISCUSSION THREAD

**Discussion by himoto** [biomass-dev/biomass#137] [Q&A] [answered]
Hi @formersbach,

https://github.com/biomass-dev/biomass/blob/a2acaa2ad78b17516d222b9bcbdcdeff5fb985a5/biomass/graph.py#L210-L212

Seems `self.graph` in line 210 is not used. Can we remove this line?

**Comment by formersbach** [comment, marked as answer]
I added this so it's possible to directly access the graph object. This makes it possible to e.g. look for sub-graphs, circles, manually add or remove elements etc.
It's not used right now, but I figured it might become useful in the future. If there are any issues with pre-commit bot or something feel free to throw it out!

**Reply by himoto** [reply]
I got it, thank you for the detailed explanation! Then, I would like to keep this.
